### PR TITLE
[ODS-4856] Remove Bundling and Minification from SandboxAdmin

### PR DESCRIPTION
**Pull request by raghuramakrishna**

### CODE REVIEW

File: Application/EdFi.Ods.SandboxAdmin/wwwroot/js/edfiadmin.auth.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿/// <reference path="globals.js" />
 /// <reference path="extensions.js" />
-/// <reference path="underscore.js" />
+/// <reference path="https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.5.2/underscore-min.js" />
```
Comment (tbmeador-dlp) [inline Application/EdFi.Ods.SandboxAdmin/wwwroot/js/edfiadmin.auth.js:3]: does this do anything? my understanding was that reference paths no longer work in newer version of Visual Studio

File: Application/EdFi.Ods.SandboxAdmin/Views/Shared/Layout.cshtml [hunk 2]
```diff
@@ -37,14 +37,18 @@
...
     <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
     <script src="//cdnjs.cloudflare.com/ajax/libs/knockout/3.3.0/knockout-min.js"></script>
     <script src="//cdnjs.cloudflare.com/ajax/libs/knockout.mapping/2.4.1/knockout.mapping.min.js"></script>
     <script src="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.4/js/bootstrap.min.js"></script>
 </head>
 <body data-auth="@isAuthenticated.ToString()">
-    <script src="~/js/bundles.min.js"></script>
+    <script src="~/js/extensions.js"></script>
+    <script src="~/js/globals.js"></script>
+    <script src="~/js/edfiadmin.auth.js"></script>
```
Comment (tbmeador-dlp) [inline Application/EdFi.Ods.SandboxAdmin/Views/Shared/Layout.cshtml:51]: I missed this in the acceptance criteria but underscore was also a part of the bundles.min.js

**Review by tbmeador-dlp — CHANGES_REQUESTED** [review]
When testing I noticed the login, and other menu items, are now missing.
image

**Review by blmeyers — CHANGES_REQUESTED** [review]

File: Application/EdFi.Ods.SandboxAdmin/wwwroot/css/Site.css [hunk 3]
```diff
@@ -76,4 +77,7094 @@
 
 #clients-table td {
     vertical-align: middle;
+}
```
[the remark this answers is not in the document]
Comment (raghuramakrishna) [reply Application/EdFi.Ods.SandboxAdmin/wwwroot/css/Site.css]: I have removed the repeated classes from css

File: Application/EdFi.Ods.SandboxAdmin/Views/Shared/Layout.cshtml [hunk 4]
```diff
@@ -37,14 +37,19 @@
 <head>
     <title>@ViewBag.Title</title>
     @RenderSection("head", required: false)
-    <link rel="stylesheet" href="~/css/site.min.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.0.0/css/flatly/bootstrap.min.css" />
```
[the remark this answers is not in the document]
Comment (raghuramakrishna) [reply Application/EdFi.Ods.SandboxAdmin/Views/Shared/Layout.cshtml:40]: Removed

File: Application/EdFi.Ods.SandboxAdmin/wwwroot/js/edfiadmin.auth.js (hunk 1 above)
Comment (raghuramakrishna) [reply Application/EdFi.Ods.SandboxAdmin/wwwroot/js/edfiadmin.auth.js]: removed reference paths

**Review by blmeyers — APPROVED** [review]

**Review by tbmeador-dlp — APPROVED** [review]

**Pull request merged**